benchmarks-log4j2/src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" packages="io.scalecube.benchmarks.log4j2">
+
+  <Properties>
+    <Property name="patternLayout">%level{length=1} %d{ISO8601} %c{1.} %m [%t]%n</Property>
+  </Properties>
+
+  <Appenders>
+    <RollingFile name="FILE"
+                 fileName="reports/benchmarks.log"
+                 filePattern="reports/benchmarks-%d{yyyy-MM-dd-HH}-%i.log.zip"
```

Comment:
Plz remove `.zip`  because otherwise log frameowrk will strive to archive a file